templates/API_TEMPLATE.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: default
+title: Example API 
+parent: 
+nav_order: 
+---
+
+# Example API 
+Introduced 1.0
+{: .label .label-purple }
+
+The Example API ... (descriptive sentence about what this API does).
```

Comment:
Is it appropriate to move this description below "The Example API"? To express it as it would appear in the documentation? (Am I imagining this correctly?)

Reply:
The description is "The Example API... " and it is already under the heading "Example API", so it's exactly how it would appear in the documentation.